mysql-test/suite/rocksdb/include/bloomfilter.inc

```diff
@@ -26,12 +26,20 @@ DELIMITER ;//
 --source $ddl
 --source suite/rocksdb/include/bloomfilter_load_select.inc
 
+--exec sed s/##CF##/" COMMENT 'rev:cf_short_prefix'"/g $tmpl_ddl > $ddl
```

Comment:
SQL string statement manipulation or perl would be more canonical for portability, but I guess it's not something to lose sleep over